README.md

```diff
@@ -30,7 +30,9 @@ The
 [`api/prometheus` directory](https://github.com/prometheus/client_golang/tree/master/api/prometheus)
 contains the client for the
 [Prometheus HTTP API](http://prometheus.io/docs/querying/api/). It allows you
-to write Go applications that query time series data from a Prometheus server.
+to write Go applications that query time series data from a Prometheus
+server. It is in an early stage and further development is currently on hold
```

Comment:
I'd be wary of including such a statement in a readme, it may be misconstrued to mean the whole client is unmaintained.